docs/api-docs/apps/guide/apps-02-types.md

```diff
@@ -3,88 +3,99 @@
 
-Some apps qualify to be listed in the BigCommerce App Marketplace as “Connector” apps. These apps use manual OAuth token creation instead of the single-click app flow.
-While we always recommend and prefer the single-click app flow’s programmatic OAuth exchange, certain use cases might not be compatible.
-
-Use Cases:
-- Customized integrations that vary per store.
-- Integrations that do not provide any content for an iframe.
-- Apps that do not provide Web services for OAuth exchange. (For example: offline sync apps.)
-- Apps that do not provide Web or cloud-hosted components for a dashboard.
-
-<div class="HubBlock--callout">
-<div class="CalloutBlock--info">
-<div class="HubBlock-content">
 
-<!-- theme:  -->
+## Single-Click
 
-### Connector apps
-> Connector apps require the store owner to create API Credentials in the Control Panel and then pass those values into the app manually. The single-click installation process is preferred since it does not require this step.
+Single-click apps are the recommended app type. They use [OAuth 2.0 Authorization Code Grant](https://oauth.net/2/grant-types/authorization-code/) (oauth2.net) flow. Users initiate installation by clicking **Install**:
 
-</div>
-</div>
-</div>
+![Install App](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-02-types-01.png "Install App")
 
-<br>
+The app requests the user to grant permissions associated to scopes configured for the app in [DevTools](https://devtools.bigcommerce.com/my/apps):
 
-To create a connector app that uses manual OAuth token creation, make the following selections on the Technical tab when registering your app in Dev Tools:
+![App Permissions](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/apps-02-types-02.png "App Permissions")
 
-1. Set the App Type option to Connector.
-2. Enter your app’s Signup URL. This is the URL where a user would begin to configure the connection between your service and the BigCommerce store.
-3. Under OAuth scopes, select the scopes that your app requires for BigCommerce review only. Later, you will need to instruct store users to create a token with the production scopes your app needs.
+Once granted, the app can request a permanent `access_token` for making REST API requests on the user's behalf.
 
-BigCommerce must review and approve your connector app proposal before we will accept the app’s submission to the Marketplace.
+## Connector
+Connector apps use manual OAuth token creation instead of the single-click app flow. Store owner's generate [Store API Credentials](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#revoking-store-api-credentials) manually and enter them into the app's configuration. While single-click apps are recommended, certain use cases might not be compatible:

```

Comment:
```suggestion
Connector apps use manual OAuth token creation instead of the single-click app flow. Store owners generate [store API credentials](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#revoking-store-api-credentials) manually and enter them into the app's configuration. While single-click apps are recommended, the following use cases might not be compatible:

```